tools/heir-opt.cpp

```diff
@@ -556,8 +560,13 @@ struct MlirToBgvPipelineOptions
       llvm::cl::init(1024)};
 };
 
-void mlirToBgvPipelineBuilder(OpPassManager &pm,
-                              const MlirToBgvPipelineOptions &options) {
+typedef std::function<void(OpPassManager &pm,
+                           const MlirToRLWEPipelineOptions &options)>
+    RLWEPipelineBuilder;
+
+void mlirToRLWEPipelineBuilderInternal(OpPassManager &pm,
```

Comment:
"Internal" is just noise here.

Reply:
Argh! My bad. Leftover from a lengthy process debugging why adding the cl::opt was not working

Reply:
done